src/components/Menu/components/AddPopover.js

```diff
@@ -0,0 +1,18 @@
+import { AddIcon } from "@chakra-ui/icons";
+import PopoverBase from "../../../common/PopoverBase";
+import CircleIconButton from "../../../common/buttons/CircleIconButton";
+import NewCategoryForm from "./NewCategoryForm";
+import NewItemForm from "./NewItemForm";
+
+const AddPopover = ({ target }) => {
+  const headerElement = target === "category" ? "Yeni kategori ekleyin" : "Yeni item ekleyin";
```

Comment:
So I think there are two targets, one is category and one is item.

These are first of all should be a constant like the following:

```js
export const popoverTarget = {
  category: 'category',
  item: 'item',
}
```

We need another constant to map the target to title:
```
export const titleMap = {
  category: 'title 1',
  item: 'title 1',
}
```

With this way, we can get the title inside the component via  {titleMap[target]}

And when we want to do an if check like target === "category", we will do target === popoverTarget.category